warehouse/models/intermediate/gtfs/_int_gtfs.yaml

```diff
@@ -175,7 +175,7 @@ models:
       - name: step
       - name: base64_url
       - name: ts
-  - name: int_gtfs_schedule__stop_times
+  - name: int_gtfs_schedule__incremental_stop_times
```

Comment:
was this just pointing at the old table?

Reply:
This was not pointing at anything, broken reference, fixed here because I noticed it